spec/unit/sneakers_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+require 'sneakers'
+require 'airbrake/sneakers'
+
+RSpec.describe Airbrake::Sneakers::ErrorReporter do
+  let(:worker) do
+    queue = instance_double('queue')
+    allow(queue).to receive_messages(
+      name: 'test-queue',
+      opts: {},
+      exchange: instance_double('exchange')
+    )
+
+    Sneakers.clear!
+    Sneakers.configure(daemonize: true, log: '/dev/null')
+    Sneakers::Worker.configure_metrics
```

Comment:
L15..L16 probably belong to a `before` block.

Reply:
valid point